styles.css

```diff
@@ -0,0 +1,197 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+main {
+  margin-top: -80px;
+  margin-left: 50px;
+  padding-right: 50px;
+}
+
+ul {
+  list-style: none;
+}
+
+li {
+  list-style: none;
+}
```

Comment:
Group selectors with the same style definition, or create a new class

Reply:
1. To minimize code group selectors with the same style definition added.
2. The bottom part styling changed.